pkg/detectors/databrickstoken/databrickstoken.go

```diff
@@ -0,0 +1,87 @@
+package databrickstoken
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
+	domain = regexp.MustCompile(`\b(https:\/\/.*\.cloud\.databricks\.com)\b`)
```

Comment:
nit: I believe you don't have to escape the `/` in the regex. We can likely just get away with: `\b(https://.*\.cloud\.databricks\.com)\b`. Additionally do you think it makes sense to restrict the subdomain to something like `[a-zA-Z0-9-]` to exclude periods?

The overall regex would be `\b(https://[a-zA-Z0-9-]+\.cloud\.databricks\.com)\b`